ToGather/ToGather/Screens/Core/Start/View/StartView.swift

```diff
@@ -10,16 +10,15 @@ import Firebase
 
 struct StartView: View {
     @EnvironmentObject var userViewModel: UserViewModel
-    @StateObject var onboardingViewModel = OnBoardingViewModel()
+    @AppStorage("isVisited") var isFirstOn = !UserDefaults.standard.bool(forKey: "isVisited")  // 처음에 자동으로 초기화된 값이 false라서 이렇게 작성함. true로 들어가도록.
```

Comment:
```suggestion
    @AppStorage("isVisited") var isVisited = UserDefaults.standard.bool(forKey: "isVisited")
```
isVisited으로 하면 문제가 있을까요?

Reply:
로직상은 문제가 없는데 코드 읽을때 헷갈려서 isFirstOn으로 했습니다! 이해하고 힘드실까요 😂

Reply:
@KimDaeSeong8721 
StartView 밑에 onAppear을 붙여서 UserDefaults.standard.setValue(true, forKey: "isFirstOn")를 추가해주면 isVisited 말고  isFirstOn으로 유저 디폴트에 저장할 수 있을 것 같습니다! 이 방법도 한번 고려해주세요!

Reply:
> @KimDaeSeong8721
> StartView 밑에 onAppear을 붙여서 UserDefaults.standard.setValue(true, forKey: "isFirstOn")를 추가해주면 유저디폴트에서 isVisited 말고 isFirstOn으로 이름을 사용할 수 있을 것 같습니다! 이 방법도 한번 고려해주세요!

해당 방법을 사용하면 앱이 구동될때마다 isFirstOn이 true가 돼서 온보딩이 진행되지 않나요?

Reply:
self.$!isVisited 자체가 문법상 어긋나서 사용을 못해서 저런식으로 작성했습니다 ㅠㅠ
그리고 onAppear에 말씀해주신 코드를 추가할경우 닐이 말씀하신대로 앱이 구동될때마다 true가 됩니다.